daemon/src/model.rs

```diff
@@ -207,6 +207,10 @@ impl BitMexPriceEventId {
             .join(&self.to_string())
             .expect("Event id can be joined")
     }
+
+    pub fn as_unix_timestamp(&self) -> i64 {
+        self.timestamp.unix_timestamp()
+    }
```

Comment:
I think I'd prefer if we expose the strongly-typed timestamp (`OffsetDatetime`) instead of the unix timestamp.

Reply:
Force pushed, please check again :)